### DISCUSSION TITLE
테이블 관련

### DISCUSSION THREAD

**Discussion by KiWooPark** [pronist/hELLO#120] [Q&A] [answered]
안녕하세요. 제작해주신 스킨을 적용해서 글을 작성하고 있는데 테이블 관련해서 질문 드립니다.

![image](https://user-images.githubusercontent.com/72461214/183489126-a6bcfdc7-94c3-4ef7-8128-ec33430f7ffc.png)

이런식으로 테이블의 셀 너비를 자동으로 각각의 셀 안에 글자 너비 만큼 조절되도록 할수있는 방법이 있을까요?

<img width="817" alt="image" src="https://user-images.githubusercontent.com/72461214/183489431-a5b6e5d4-3d26-40aa-8dfc-cc190a4bdcb9.png">

**Comment by pronist** [comment, marked as answer]
@KiWooPark 테이블 사이즈의 경우 글을 쓰는 시점 에디터 상에서 수동으로 변경할 것을 권하고 있지만, 코드 변경을 통해 CSS 및 Javascript 를 추가하여 테이블의 셀 개수를 계산해서 그에 따라 백분율로 처리하면 가능할 것입니다. 이를테면 열이 5 인 경우 전체 사이즈를 100으로 간주하고 100 / 5 와 같이 처리하여 각 셀은 20% 로 처리합니다.

**Reply by KiWooPark** [reply]
감사합니다. 
테이블의 경우 CSS에서 테두리 생기도록 수정하였고, HTML로 작성해서 사용하니 원하는 모양으로 나오네요!
